lte/gateway/c/core/oai/lib/n11/amf_client_proto_msg_to_itti_msg.cpp

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <iostream>
+#include <string>
+
+#include "3gpp_33.401.h"
+#include "amf_app_messages_types.h"
+#include "amf_client_proto_msg_to_itti_msg.h"
+#include "common_types.h"
+#include "security_types.h"
+
+extern "C" {}
+
+using magma::lte::M5GAuthenticationInformationAnswer;
+
+namespace magma5g {
+
+void convert_proto_msg_to_itti_m5g_auth_info_ans(
```

Comment:
can you add test coverage?

Reply:
Test coverage added in (lte/gateway/c/core/oai/test/n11/test_auth_service_client.cpp)

Reply:
You need to compare against an expected struct in the testing. What you did is not covering the whole structure.

Reply:
The Unit Test will be covered in next PR.